lib/api/controllers/securityController.js

```diff
@@ -19,6 +19,66 @@ var
  */
 function SecurityController (kuzzle) {
   /**
+   * Get the role mapping
+   *
+   * @returns {Promise}
+   */
+  this.getRoleMapping = function securityGetRoleMapping () {
+    return kuzzle.internalEngine.getMapping({index: kuzzle.internalEngine.index, type: 'roles'})
+      .then(response => ({mapping: response[kuzzle.internalEngine.index].mappings.roles.properties}));
+  };
+
+  /**
+   * Update the roles collection mapping
+
+   * @param {Request} request
+   * @returns {Promise}
+   */
+  this.updateRoleMapping = function securityUpdateRoleMapping (request) {
```

Comment:
Yep, I'm waiting the merge of you're PR #616 because you changed the `assert*` signatures there